polars/polars-lazy/polars-plan/src/dsl/function_expr/datetime.rs

```diff
@@ -21,6 +21,8 @@ pub enum TemporalFunction {
     Microsecond,
     Nanosecond,
     TimeStamp(TimeUnit),
+    Truncate(&str, &str),
```

Comment:
I'm a little stuck on why the enum doesn't allow string slices as parameters. I have seen in string.rs it uses String as the parameter type but then in the impl it still uses the slice, but when I tried this the compiler complains about expecting a slice.

Reply:
The enum should have Strings. They own the data.